ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -447,8 +447,12 @@ TorchCore::processIValue (
  */
 int
 TorchCore::serializeOutput (
-    const torch::jit::IValue &value, GstTensorMemory *output, unsigned int *idx)
+    const torch::jit::IValue &value, GstTensorMemory *output, unsigned int *idx, unsigned int limit_idx)
```

Comment:
To fix clang format error, please update as below:
```suggestion
TorchCore::serializeOutput (const torch::jit::IValue &value,
    GstTensorMemory *output, unsigned int *idx, unsigned int limit_idx)
```
and you can check error [here](http://ci.nnstreamer.ai/nnstreamer/ci/repo-workers/pr-checker/4077-202305101037130.039629936218262-35bc7428d504c033ed0364f27a86c3e824837bd5/report/clang-format.patch)

And please check DCO and add the sign-off: [DCO](https://github.com/apps/dco)

Reply:
Thanks @gichan-jang! I took care of those two issues. There are still two checks failing and I would need some advise how to address those.

Reply:
The test failure is irrelevant to your change. 
I started the TAOS-CI again and it passed now.